src/screens/InputUsername/index.js

```diff
@@ -349,7 +350,9 @@ const styles = StyleSheet.create({
   input: {
     borderWidth: dimen.normalizeDimen(1),
     borderRadius: dimen.normalizeDimen(8),
-    borderColor: COLORS.silver,
+    borderColor: COLORS.gray200,
+    backgroundColor: COLORS.gray100,
+    color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The border color, background color, and text color of the input field have been updated. Make sure these changes don't affect readability and accessibility.
```diff
- borderColor: COLORS.silver,
+ borderColor: COLORS.gray200,
+ backgroundColor: COLORS.gray100,
+ color: COLORS.white2,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->